autoload/clang_format.vim

```diff
@@ -209,6 +209,7 @@ function! clang_format#replace(line1, line2)
     let sel_save = &l:selection
     let &l:selection = 'inclusive'
     let [save_g_reg, save_g_regtype] = [getreg('g'), getregtype('g')]
+    let [save_unnamed_reg, save_unnamed_regtype] = [getreg(''), getregtype('')]
```

Comment:
Yes, it looks clearer using v:register.

I changed it and confirmed it works ok.